applications/IgaApplication/custom_conditions/sbm_laplacian_condition.h

```diff
@@ -291,6 +283,13 @@ namespace Kratos
         Parameters ReadParamatersFile(
         const std::string& rDataFileName) const;
 
+        // sbm variables
+        array_1d<double, 3> normal_parameter_space;
+        Matrix H_sum = ZeroMatrix(1, this->GetGeometry().size());
+        Vector d;
+        std::vector<Matrix> mShapeFunctionDerivatives;
+        IndexType basis_functions_order;

```

Comment:
```suggestion
        array_1d<double, 3> mNormalParameterSpace;
        Matrix mHsum = ZeroMatrix(1, this->GetGeometry().size());
        Vector mD;
        std::vector<Matrix> mShapeFunctionDerivatives;
        IndexType mBasisFunctionOrder;

```
Please follow the style guide and naming convention. Besides, a more self-descriptive name should be used for d (e.g., `mDistanceVector`)